cookies/views/resource.py

```diff
@@ -126,6 +126,9 @@ def resource_list(request):
     Display all :class:`.Resource` instances to which the user has access.
     """
 
+    # Check for removing empty text. Find and delete entries which do not have a primary id
+    ResourceContainer.objects.filter(primary_id=None).delete()
```

Comment:
This approach is pretty unsafe. What if someone goes to the resource page right after someone uploaded a file before submitting the metadata? This needs to be solved in a different way.